classes/PodsAPI.php

```diff
@@ -4891,13 +4891,13 @@ private function export_pod_item_level( $pod, $params ) {
 				$field                = $pod->fields[ $field['name'] ];
 				$field['lookup_name'] = $field['name'];
 
-				if ( in_array( $field['type'], $tableless_field_types ) && ! in_array( pods_var( 'pick_object', $field ), $simple_tableless_objects ) ) {
+				if ( in_array( $field['type'], $tableless_field_types ) && ! in_array( pods_v( 'pick_object', $field ), $simple_tableless_objects ) ) {
```

Comment:
* :warning: Not using strict comparison for in_array; supply true for third argument.
* :warning: Not using strict comparison for in_array; supply true for third argument.

<sub>hash: c539da1f0cf5937670c9b39ddbf1e4c0</sub>